pkg/virt-controller/services/template.go

```diff
@@ -1137,6 +1139,8 @@ func getNamespaceAndNetworkName(namespace string, fullNetworkName string) (strin
 	return precond.MustNotBeEmpty(namespace), fullNetworkName
 }
 
+type TemplateServiceOption func(service *templateService)
```

Comment:
There is no need to expose it, `templateServiceOption` should be enough.
There is also no need to name the argument of the function, the type is informative enough.

Reply:
Done